DM-1624 Loriot Whats New 1016 Update (#75)
SoftwareAG/c8y-releasenotes: content/release-10-16-0/whatsnew-10-16-0.md

---

Release 10.16.0 includes the following new features or major feature enhancements.

> ### Multiple LNS Connections for Loriot

Reviewer: ```suggestion
### Multiple LNS connections for Loriot
```

The Loriot agent now supports the configuration of multiple connections per tenant. One connection can be associated with multiple devices. Each connection can be updated and deleted.

![All devices](/images/release-notes/multiple_lns_connection_loriot.png)

> For details, refer to [Loriot Multiple LNS Connection](https://cumulocity.com/guides/protocol-integration/lora-loriot/#configure-loriot-credentials-cumulocity)

Reviewer: ```suggestion
For details, refer to [Device registration via Cumulocity IoT](https://cumulocity.com/guides/protocol-integration/lora-loriot/#configure-loriot-credentials-cumulocity).
```

> ### Support for Loriot Device Registration

Reviewer: ```suggestion
### Support for Loriot device registration
```

> Using the Loriot LNS Connections created in the **Connectivity** page in **Settings** under **Administration**, devices can now also be registered using the **Device Registration** page. 

Reviewer: ```suggestion
Using the Loriot LNS connections created in the **Connectivity** page in **Settings** under **Administration**, devices can now also be registered using the **Device Registration** page. 
```

![Device Registration](/images/release-notes/loriot-registration.png)

> For Details refer to [Loriot Device Registration](https://cumulocity.com/guides/protocol-integration/lora-loriot/#configure-loriot-credentials-cumulocity)

Reviewer: ```suggestion
For details refer to [Device registration via Cumulocity IoT](https://cumulocity.com/guides/protocol-integration/lora-loriot/#configure-loriot-credentials-cumulocity).
```

> For the devices registered with the Loriot agent, downlink operations can be sent from the **Shell** tab

Reviewer: ```suggestion
For the devices registered with the Loriot agent, downlink operations can be sent from the **Shell** tab.
```

> For Details refer to [Loriot Downlink Operations](https://cumulocity.com/guides/protocol-integration/lora-loriot/#operations-loriot)

Reviewer: ```suggestion
For details refer to [Sending operations](https://cumulocity.com/guides/protocol-integration/lora-loriot/#operations-loriot).
```

Reviewer: @ashhSAG93 Please make sure to use the exact headings in links.

Author: Sure @BeateRixen we will check the links and make the changes
